ci/kind/kind-setup.sh

```diff
@@ -663,7 +663,6 @@ while [[ $# -gt 0 ]]
       shift 2
       ;;
     --flexible-ipam)
-      add_option "--flexible-ipam" "create"
```

Comment:
I suppose the related script in the Jenkins YAML should also be updated?

Reply:
yes, i am working on backport PR:#7139 after that we don't need jenkins testbed for ipam.